src/node/config/excludedOperations.mjs

```diff
@@ -0,0 +1,28 @@
+/**
+ * Operations to exlude from the Node API
+ *
+ * @author d98762656 [d98762625@gmail.com]
+ * @copyright Crown Copyright 2018
+ * @license Apache-2.0
+ */
+export default  [
+    "Fork",
+    "Merge",
+    "Jump",
+    "ConditionalJump",
+    "Label",
+    "Comment",
+
+    // esprima doesn't work in .mjs
+    "JavaScriptBeautify",
+    "JavaScriptMinify",
+    "JavaScriptParser",
+
+    // Relies on state of recipe.
+    // "Magic",
+
+    "RenderImage",
+    "SyntaxHighlighter",
+
+    "DetectFileType",
```

Comment:
Does this work now that it has been rewritten upstream?

Reply:
Yes! 😄